libs/sql-schema-describer/src/lib.rs

```diff
@@ -397,6 +397,8 @@ pub struct ForeignKey {
     pub referenced_columns: Vec<String>,
     /// Action on deletion.
     pub on_delete_action: ForeignKeyAction,
+    /// Action on deletion.
+    pub on_update_action: ForeignKeyAction,
```

Comment:
Comment is wrong.

Reply:
Fixed in https://github.com/prisma/prisma-engines/pull/1108/commits/b6d384652d21f6f394527c4438fe335835e8a679